### ISSUE TITLE
dist/ does not contain License for distributed binaries (including XQEMU itself)

### ISSUE THREAD

**Issue by JayFoxRox** [xqemu/xqemu#85]
dist/ only includes our exe files and dependencies in dll form, but not any documentation or legal information.
This makes it a violation of a handful of licenses probably.

**Comment by GXTX** [comment]
This is correct, currently in violation of GPL by not including it in the distributables.
